runtime/src/iree/hal/drivers/hip/event_pool.c

```diff
@@ -251,15 +251,33 @@ iree_status_t iree_hal_hip_event_pool_acquire(
   // Allocate the rest of the events.
   if (remaining_count > 0) {
     IREE_TRACE_ZONE_BEGIN_NAMED(z1, "event-pool-unpooled-acquire");
```

Comment:
Zone names should always be proper C-like function names - if you need additional information then you can use e.g. `IREE_TRACE_ZONE_APPEND_TEXT(z0, "unpooled acquire");`

In tracy those show up under the parent zone as searchable/groupable text. If you did want to keep it a zone (which makes it trickier to manage as here) then it should be something like `iree_hal_hip_event_pool_acquire_unpooled` (and to simplify things you can just make that a function instead of doing it inline and then it has its own trace zone in the standard form instead of nested inside a loop).

Reply:
This is code that predates me, so I have no strong opinion here :)
I will append the text.